anti_patterns/deny-warnings.md

```diff
@@ -36,7 +36,10 @@ All this conspires to potentially break the build whenever something changes.
 
 Furthermore, crates that supply additional lints (e.g. [rust-clippy]) can no
 longer be used unless the annotation is removed. This is mitigated with
-[--cap-lints].
+[--cap-lints]. The `--cap-lints=warn` command line argument, turns all `deny`
+lint errors into warnings. But be aware that `forbid` lints are stronger than
+`deny` hence the 'forbid' level can not be overridden to be anything lower than
```

Comment:
```suggestion
`deny` hence the 'forbid' level cannot be overridden to be anything lower than
```

I am not native English speaker, so I don't know if I am right here.

Reply:
good spot